support of uploading native symbols and source code (#1497)
getsentry/sentry-docs: src/collections/_documentation/platforms/android/index.md

    // during a build.  If you disable this, you'll need to manually
    // upload the mapping files with sentry-cli when you do a release.
    autoUpload true

    // Disables or enables the automatic configuration of Native Symbols
    // for Sentry. This executes sentry-cli automatically so
    // you don't need to do it manually.
    // Default is disabled.
    uploadNativeSymbols false

>     // Includes or not the source code of native code for Sentry.

Reviewer: ```suggestion
    // Does or doesn't include the source code of native code for Sentry.
```

}
```

> If you want to use the SDK with the NDK but you still want to support the devices on the API level lower than 16 you can update your manifest as follows:

Reviewer: ```suggestion
If you want to use the SDK with the NDK, but you still want to support the devices on the API level lower than 16, you can update your manifest as follows:
```

```xml
    <!-- Merging strategy for the imported manifests -->
      <uses-sdk
            tools:overrideLibrary="io.sentry.android" />
```

> With these changes, the NDK integration is going to be used only on the devices with the API level ≥ 16 and the rest of the devices with API level ≥14, but ≤16 will use just the SDK.

Reviewer: I tried making this a bit more digestible. Hoping it's still valid.
```suggestion
With these changes, the NDK integration will be used only on:
- the devices with the API level ≥ 16  
- the rest of the devices with API level ≥ 14

Any device ≤ 16 will use the SDK.
```

Author: we (@fdrabek too) simplified a little bit, hopefully, it's good to go :)
